index.js

```diff
@@ -499,6 +519,37 @@ const getPhpVersionVulnerabilitiesOfGitHubRepo = async (githubToken, repo, versi
   return vulnerabilities.filter(vulnerability => vulnerability !== undefined)
 }
 
+/**
+ * Report any use of an End-Of-Life'd (EOL'd) PHP version in the specified
+ * GitHub repos.
+ *
+ * @param {string} githubToken - A GitHub personal access token
+ * @param {string[]} repos - A list of repo names
+ * @param {string} versionsCsvUrl -- The URL where we can retrieve a CSV file of
+ *     the PHP versions used by various Docker images.
+ * @returns {Promise<{repo, detailsUrl, ecosystem, identifiers, inspectedOn, packageName, severity, summary, vulnerableVersionRange}[]>}
+ */
+const getPhpVersionVulnerabilitiesOfGitHubRepos = async (
+  githubToken,
+  repos,
+  versionsCsvUrl
+) => {
+  const results = []
+  for (let i = 0; i < repos.length; i++) {
```

Comment:
since you are just doing some work and not modifying repos, I think you could use .forEach here.

Reply:
I may be misunderstanding exactly what would happen if I used a `.forEach()`, but in part I want to ensure that we check these one at a time, waiting for each to finish before checking the next, to reduce the chances of hitting a rate-limit on the GitHub API. However, I have now updated this to use a `for/of` rather than the long `for` syntax.